### ISSUE TITLE
Plugins are red and cannot be checked in the load order modal

### ISSUE THREAD

**Issue by Dawnx2** [matortheeternal/ztools#16]
https://imgur.com/BqjNhDU

**Comment by matortheeternal** [comment]
It looks like some kind of exception happened because the plugins don't have their load orders (the empty brackets).  Would you be willing to press Ctrl+Shift+I to open the developer console and post a screenshot of what the last console message is?

Also, regarding the red and disabled item: this is intended behavior when a required master is not loaded before the plugin.  It indicates the plugin cannot be loaded.  In this case it may be happening because zEdit read your load order incorrectly (it may also be related to the aforementioned exception).

**Comment by Dawnx2** [comment]
https://imgur.com/a/Ahszs

**Comment by matortheeternal** [comment]
Should be fixed, I'll get a developer build to you in just a moment.

**Comment by matortheeternal** [comment]
@Dawnx2 try the latest v0.1.1 release.  It should fix this issue for you.  :smile:

**Comment by matortheeternal** [comment]
@Dawnx2 Found out what the problem actually was, try [this hotfix](http://puu.sh/xA4sp.7z) instead.

I did some research with some helpful folks in the modding tools Discord and it turns out the problem was zEdit was doing case sensitive comparison when checking if master plugins were available.  Turns out xEdit (and the game) use case insensitive comparison, so that was causing plugins to be disabled in the load order modal when they shouldn't have been.
